package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "PtushkiMobile",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest",
+    "lint": "eslint ./src/**/*.js App.js",
+    "lint:fix": "eslint ./src/**/*.js App.js --fix"
+  },
+  "dependencies": {
+    "react": "16.8.3",
```

Comment:
readux and redux-saga will be added later?

Reply:
added only core dependencies, also, it seems like there would be no need in redux since we have device storage